docs/source/services/timelock_service/reverse-migration.rst

```diff
@@ -0,0 +1,140 @@
+.. _timelock-reverse-migration:
+
+Reverse Migration
+=================
+
+.. danger::
+
+   AtlasDB does not natively support reverse migrations. Improperly executing reverse migration from external timestamp
```

Comment:
I changed this from the original "does not support" because that wasn't true. I think I'll remove the first sentence altogether.

Yes, your understanding is correct (though we've already said that in the last sentence here).